droidlet/dashboard/web/src/components/AnnotationComponents/PolygonTool.jsx

```diff
@@ -64,18 +64,27 @@ class PolygonTool extends React.Component {
   }
 
   render() {
+    let imageSize = "500px"; // default is 500px for the web dashboard
+    if (this.props.imageWidth) {
+      imageSize = this.props.imageWidth;
+    }
     return (
       <div>
         <p>Please trace the {this.props.object}</p>
         <canvas
           ref={this.canvasRef}
-          width="500px"
-          height="500px"
+          width={imageSize}
+          height={imageSize}
           tabIndex="0"
           onClick={this.onClick}
           onMouseMove={this.onMouseMove}
           onKeyDown={this.keyDown}
         ></canvas>
+        {this.props.isMobile && (
+          <button onClick={this.enterButtonMobile.bind(this)}>
+            Finished annotating the {this.props.object}
```

Comment:
This sounds more like a confirmation message than a button name. Maybe `Submit object annotations`? 

Also, in the demo video there was "Finished Annotation" and "Finished annotating objects" - guessing this is final?

Reply:
So I found Submit object annotations a bit misleading, since it's really not 'submitting'. The user can submit many annotation when submitting "one annotation." IE, in the same annotation, a user can label a chair and a table. Once the user finishes labeling the chair, the user will click on the button in question, then the user can either submit the annotation with 1 object, or label another object. Maybe I can rename this "Finished labeling {this.props.object}"

And regarding the demo, the demo was taken before I decided to change the string (so it would be personalized to the actual object).